en/docs/get-started/how-to-guides/ai-code-generation.md

```
@@ -157,31 +174,44 @@ You will now see the projects listed in the **Project Explorer**.
+        Before running the API, update the generated configurations with real credentials:
+
+          - Update the email connection with the credentials of the sender's email account.
+          - Update the `API_KEY` variable with your OpenWeather API key.
+
+        You can ask the Copilot to open a specific file for you — for example, *"Open the Email connection so I can update the credentials."*
+
+### Step 4: Build and run with the Copilot
+
+The Copilot can build the project and start the MI runtime for you — no need to leave the chat panel. Send a follow-up message:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)